### DISCUSSION TITLE
HOW TO TAKE IMAGES MEGATHREAD/MAKE & USE NEW DECKS

### DISCUSSION THREAD

**Discussion by paperc07** [AdriaGual/marvel-snap-bot#94] [General]
1. There are 2 ways you can go about this one is to open your Info.py file and set the fields shown below to True from False
<img width="669" alt="Untitled" src="https://user-images.githubusercontent.com/9095222/219981456-79d5df6c-08de-41ad-9856-ce2b18007aae.png">

2. Your next option is to leave it at False and to click on one of these 2 files (print_fields.py or print_hand_cards.py) as you see fit. If you want to print the fields you would double click on the print_fields.py file and you would get something like this 
<img width="708" alt="2023-02-19 18_25_23-Error when opening Hand_cards py · Issue #88 · AdriaGual_marvel-snap-bot" src="https://user-images.githubusercontent.com/9095222/219981810-6191ae64-d097-4d66-89d4-405cf8c543cb.png">
<img width="490" alt="py_EZdrgtuuQZ" src="https://user-images.githubusercontent.com/9095222/220004762-2c56de33-5b54-4132-bd22-ca1582a7891b.png">

3. Now that you have your fields pulled up (you don't need all 3) but I gave you an example with 3 and then the next one with just the one location. Then what you want to do is snip the location name using the snipping tool (on windows pc, hit START type in Snip and it should come up) and cut out the location name of the screenshot. Here is a video of the process. I WOULD RECOMMEND KEEPING THE SCREENSHOT JUST INCASE THE LOCATION DOESN'T READ YOU CAN TAKE ANOTHER SNIP OR 2 AND FIGURE OUT WHICH ONE WORKS. I USE SHARE X FOR MY SCREENRECORD AND SCEENSHOTS

https://user-images.githubusercontent.com/9095222/220004571-71b2da9f-3793-4c98-8e93-15bd353ada9e.mp4

4. This is what the end product will look like <img width="78" alt="ice box" src="https://user-images.githubusercontent.com/9095222/219997005-b0e2730d-52ec-4683-af55-b74c8b0a8f64.png">

5.  Now ideally you should be able to just throw this into your Marvel_snap_bot\images\fields folder mine looks like this (C:\Users\paper\marvel-snap-bot\images\fields) your's will obviously be different, But essentially you should be able to name it the_ice_box.png and drop it in there like this 
<img width="655" alt="explorer_BCkNGB6PLP" src="https://user-images.githubusercontent.com/9095222/219998054-fcd27e39-a309-4968-96dc-7cfdbc7dcf74.png">

6. Then you need to go edit your field_list.py file and add it in there like this 
<img width="486" alt="notepad++_MwVIDHArRE" src="https://user-images.githubusercontent.com/9095222/219998492-0611e776-3fdb-4701-a1c9-480ae1483b9d.png">

7. Now in a perfect world that would be all you need to be able to make locations work and honestly if it was that easy you wouldn't even need this guide cause I would have spent a few hours getting all the images for you and you would just simply have to download them all and drop them all in the images\fields folder...... But it's not a perfect world so im teaching you the way and hopefully you can get it to work until I find a better way to make every single location work ANYWAYS MOVING ON!!!

8. Next we have actual screenshots and snips of our hand cards which we are now going to be double clicking on the print_hand_cards.py file and it will look like this  PLEASE USE THE DEFAULT CARDS AND NOT YOUR FANCY VARIANTS UNLESS YOU REALLY WANT TO TAKE SCREENSHOTS OF YOUR VARIANTS SO THEY WORK BUT ALSO IF YOU GET THEM WORKING BE SO KINDEST TO UPLOAD THEM HERE FOR OTHER USERS TO USE 
<img width="755" alt="71NuqV2cRJ" src="https://user-images.githubusercontent.com/9095222/219999341-9d66bf67-3818-4f0e-ba54-145aac62dcfe.png">

9. Now in this video im not swearing this is 100% going to work, but according to the images in the (C:\Users\paper\marvel-snap-bot\data) folder you can see let's use ant man for example look at his images and it's all of him, his head, his stomach area and some of his logo. I am under the impression you just need a few different images, cause the bot is picky on identifying the characters.
<img width="800" alt="explorer_HDSMRMq5m8" src="https://user-images.githubusercontent.com/9095222/220001208-e9dfb990-63bd-44e9-8605-777a66c9a370.png">

VIDEO 

https://user-images.githubusercontent.com/9095222/220002570-7e750228-75cf-4937-a146-9afe89640b62.mp4

10. Here is the images and inserting them into the data\quicksilver folder of the character I snipped the images from the screenshot.

<img width="80" alt="qs4" src="https://user-images.githubusercontent.com/9095222/220002107-06de00d1-c8ab-443a-9468-8f08918437ac.png">
<img width="62" alt="qs" src="https://user-images.githubusercontent.com/9095222/220002108-2f496d89-48a4-403b-8b9b-4ce2cd9d4ffe.png">
<img width="41" alt="qs1" src="https://user-images.githubusercontent.com/9095222/220002109-e87031c8-748c-4896-9442-0febedfb76e0.png">
<img width="70" alt="qs2" src="https://user-images.githubusercontent.com/9095222/220002111-491e9f38-19db-4828-9254-15dc5f49a4d5.png">
<img width="50" alt="qs3" src="https://user-images.githubusercontent.com/9095222/220002112-353a35a2-54d2-4a40-8f10-bd1c92388d6f.png">

<img width="375" alt="explorer_LumCsaQbyS" src="https://user-images.githubusercontent.com/9095222/220002377-9f817e3a-f9af-4d9e-8752-cae3449cd77b.png">

**Comment by paperc07** [comment]
@AdriaGual Let me know if I am missing anything or what I can add to make this better thank you

**Reply by AdriaGual** [reply]
That's an amazing tutorial to be honest, I don't think I could explain it better!

**Reply by paperc07** [reply]
I tried to add videos and pictures. I know that makes it way easier for people and just wasn't sure if I left anything out but now that it's done I have to go try and test out alot and figure out why locations aren't saving, why is bot randomly closing, priorities, etc.
